walkthroughs/week-4-libraries/blobstore/examples/hello-world-fetch/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -23,19 +23,25 @@
 import javax.servlet.http.HttpServletResponse;
 
 /**
- * When the fetch() function requests the /blobstore-upload-url URL, the content of the response is
- * the URL that allows a user to upload a file to Blobstore. If this sounds confusing, try running a
- * dev server and navigating to /blobstore-upload-url to see the Blobstore URL.
+ * When the fetch() function requests the /blobstore-upload-url URL, the content
```

Comment:
Did you mean to include changes to the example files in this PR?

Reply:
Yes this was intentional. When running the example I found a bug when using the back arrow so I first fixed this issue in the example and then used this as a reference.